angular.json

```diff
@@ -93,6 +94,7 @@
               "src/assets"
             ],
             "styles": [
+              "@angular/material/prebuilt-themes/deeppurple-amber.css",
```

Comment:
You don't usually check css in unit tests (this is the "test" target) - was this recommended by material?

Reply:
If it was added automatically - let's keep it, maybe it's used for something then.